lib/mail/elements/received_element.rb

```diff
@@ -28,6 +28,7 @@ def datetime_for(received)
         ::DateTime.parse("#{received.date} #{received.time}")
       rescue ArgumentError => e
         raise e unless e.message == 'invalid date'
+        warn "WARNING: Invalid date field for recieved element (#{received.date} #{received.time}): #{e.class}: #{e.message}"
```

Comment:
received*

Reply:
Oops, thanks for pointing that out. Resolved in ce905c2